librz/core/cmd_type.c

```diff
@@ -729,6 +725,76 @@ static void set_offset_hint(RzCore *core, RzAnalysisOp *op, const char *type, ut
 	}
 }
 
+RZ_API void rz_core_list_loaded_typedefs(RzCore *core, const char *input, Sdb *TDB, RzOutputMode mode) {
```

Comment:
TDB is not needed.
If you see where this is called, TDB is just `	Sdb *TDB = core->analysis->sdb_types;`, so just take it from the `core` argument, no need for an additional arg.